nexus/db-queries/src/db/datastore/mod.rs

```diff
@@ -815,14 +880,70 @@ mod test {
         logctx.cleanup_successful();
     }
 
+    #[tokio::test]
+    /// Ensure the [`RegionAllocationStrategy::RandomWithDistinctSleds`]
+    /// strategy fails when there aren't enough distinct sleds.
+    async fn test_region_allocation_strat_random_with_distinct_sleds_fails() {
+        let logctx = dev::test_setup_log("test_region_allocation");
```

Comment:
nit log name

Reply:
thanks for catching these, i hadnt even noticed the log initialization was taking this string at all